packages/enhanced/src/lib/container/runtime/FederationRuntimeModule.ts

```diff
@@ -27,10 +30,36 @@ class FederationRuntimeModule extends RuntimeModule {
    * @returns {string | null} runtime code
    */
   override generate() {
+    let matcher: string | boolean = false;
+    if (this.compilation && this.chunk) {
```

Comment:
to not rely on passthrough mock chunk callback on chunk handler resolve fail - i need to have this logic from commonjs runtime module, dynamic filesystem runtime module - which creates a matcher for chunks the current runtime module should skip. Like virtual chunk from sharing or consume module. 
With this - we should not require other parts of node and would not need to call chunk handler with mock module